### DISCUSSION TITLE
这个是什么问题

### DISCUSSION THREAD

**Discussion by cbb2625274797** [eric2788/vup_monitors#26] [Q&A] [answered]
![屏幕截图 2022-07-13 232840](https://user-images.githubusercontent.com/87512680/178773060-6d9b63a3-5ea4-4004-a47e-efd8c2e87196.png)

找不到指令XXX的处理，已掠过

**Comment by eric2788** [comment, marked as answer]
普通的消息提醒，沒有問題
或者我應該把這個放到debug消息比較好
